editor.planx.uk/src/ui/shared/ChecklistItem/ChecklistItem.tsx

```diff
@@ -51,9 +53,32 @@ export default function ChecklistItem({
         variant={variant}
         disabled={disabled}
       />
-      <Label variant="body1" className="label" component={"label"} htmlFor={id}>
-        {label}
-      </Label>
+      {description ? (
+        <Box>
+          <Label
+            variant="body1"
+            color="text.primary"
+            className="label"
+            component={"label"}
+            htmlFor={id}
+            pb={0}
+          >
+            {label}
+          </Label>
+          <Typography variant="body2" color="text.secondary" px={1.5}>
+            {description}
+          </Typography>
+        </Box>
+      ) : (
+        <Label
+          variant="body1"
+          className="label"
+          component={"label"}
+          htmlFor={id}
+        >
+          {label}
+        </Label>
+      )}
```

Comment:
I'm not 100% sure of the `htmlFor` markup here - as I understand this only the first/primary label is truly associated with the checkbox itself, and the description might be a bit adrift ? 

I looked at `DescriptionRadio` for reference and was surprised to see there wasn't any use of `htmlFor` there ! 

Let's make sure this new "description" variation is covered in our a11y test flow for upcoming audit :heavy_plus_sign:

Reply:
So I think the way this should work is that the label needs an `aria-describedby` which points to the `id` property of the description element added here 👍 

But yep, this is a good one to include on the upcoming test!